r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -1213,3 +1224,25 @@ test_that("as.difftime()", {
       collect()
   )
 })
+
+test_that("`decimal_date()` and `date_decimal()`", {
+  test_df <- tibble(
+    a = c(2007.38998954347, 1970.77732069883, 2020.96061799722,
+          2009.43465948477, 1975.71251467871, NA),
+    b = as.POSIXct(
+      c("2007-05-23 08:18:30", "1970-10-11 17:19:45", "2020-12-17 14:04:06",
+        "2009-06-08 15:37:01", "1975-09-18 01:37:42", NA)
+    )
```

Comment:
Could we also add a test case that has `Date`s as inputs?

Reply:
Done